### PR TITLE
Implement frontend for the deploy application view

### PR DESCRIPTION
**Pull request by bryk**
This is a simple end-to-end implementation for to make sure that the setup works.

### CODE REVIEW

File: src/app/frontend/deploy/deploy.html [hunk 1]
```diff
@@ -17,6 +17,17 @@
 <div layout="column" layout-padding layout-align="center center">
   <md-whiteframe class="kd-deploy-whiteframe md-whiteframe-5dp" flex flex-gt-md>
     <h3 class="md-headline">Deploy a Containerized App</h3>
-    <span>Lorem ipsum...</span>
+    <form name="userForm">
+	    <md-input-container class="md-block">
```
Comment (PeWu) [inline src/app/frontend/deploy/deploy.html]: indentation

**Comment by PeWu** [comment]
LGTM

File: src/app/frontend/deploy/deploy.html (hunk 1 above)
Comment (bryk) [inline src/app/frontend/deploy/deploy.html]: Thanks, done.

**Comment by bryk** [comment]
Done, merging.

**Pull request merged**